Azure docs updates - Import (#20903)
pingcap/docs: tidb-cloud/import-parquet-files.md

---
title: Import Apache Parquet Files from Cloud Storage into TiDB Cloud Dedicated
summary: Learn how to import Apache Parquet files from Amazon S3, GCS, or Azure Blob Storage into TiDB Cloud Dedicated.
---

# Import Apache Parquet Files from Cloud Storage into TiDB Cloud Dedicated

> This document describes how to import Apache Parquet files from Amazon Simple Storage Service (Amazon S3), Google Cloud Storage (GCS), or Azure Blob Storage into TiDB Cloud Dedicated. You can import Parquet files that are either uncompressed or compressed with [Google Snappy](https://github.com/google/snappy) (no other Parquet codecs are supported).

Reviewer: ```suggestion
This document describes how to import Apache Parquet files from Amazon Simple Storage Service (Amazon S3), Google Cloud Storage (GCS), or Azure Blob Storage into TiDB Cloud Dedicated. You can import Parquet files that are either uncompressed or compressed with [Google Snappy](https://github.com/google/snappy) (other Parquet compression codecs are not supported).
```

# Import Apache Parquet Files from Cloud Storage into TiDB Cloud Dedicated

> This document describes how to import Apache Parquet files from Amazon Simple Storage Service (Amazon S3), Google Cloud Storage (GCS), or Azure Blob Storage into TiDB Cloud Dedicated. You can import Parquet files that are either uncompressed or compressed with [Google Snappy](https://github.com/google/snappy) (other Parquet compression codecs are not supported).

Reviewer 2: ```suggestion
This document describes how to import Apache Parquet files from Amazon Simple Storage Service (Amazon S3), Google Cloud Storage (GCS), or Azure Blob Storage into TiDB Cloud Dedicated. You can import Parquet files that are either uncompressed or compressed with [Google Snappy](https://github.com/google/snappy). Other Parquet compression codecs are not supported.
```

    When you enter the source file URI and name in **Source File URIs and Names**, make sure it is in the following format `https://[account_name].blob.core.windows.net/[container_name]/[data_source_folder]/[file_name].parquet`. For example, `https://myaccount.blob.core.windows.net/mycontainer/data-ingestion/TableName.01.parquet`.

    You can also use wildcards to match the source files. For example:

    - `https://[account_name].blob.core.windows.net/[container_name]/[data_source_folder]/my-data?.parquet`: all Parquet files starting with `my-data` followed by one character (such as `my-data1.parquet` and `my-data2.parquet`) in that folder will be imported into the same target table.

    - `https://[account_name].blob.core.windows.net/[container_name]/[data_source_folder]/my-data*.parquet`: all Parquet files in the folder starting with `my-data` will be imported into the same target table.

    Note that only `?` and `*` are supported.

    > **Note:**
    >
    > The URI must contain the data source folder.

> 6. Click **Start Import**.

Author: The suggested text seems identical.

</SimpleTab>

> When you run an import task, if any unsupported or invalid conversions are detected, TiDB Cloud terminates the import job automatically and reports an importing error. You can visualize details in the **Status** field.

Reviewer: ```suggestion
When you run an import task, if any unsupported or invalid conversions are detected, TiDB Cloud terminates the import job automatically and reports an importing error. You can view details in the **Status** field.
```
